packages/ag-grid-theme/src/examples/ProvidedCellEditors.tsx

```diff
@@ -24,7 +24,7 @@ const dateString = {
       string
     >,
   ) => (params.value == null ? "" : params.value),
-  dataTypeMatcher: (value: any) =>
+  dataTypeMatcher: (value: string) =>
     typeof value === "string" && !!value.match("\\d{2}/\\d{2}/\\d{4}"),
```

Comment:
```suggestion
    !!value.match("\\d{2}/\\d{2}/\\d{4}"),
```

I thought about removing `typeof value === "string"` after changing the type of `value` to `string`, but that could cause runtime errors for consumers doing unsafe things like `dataTypeMatcher(input as string)`.